unit-tests/run-unit-tests.py

```diff
@@ -1,19 +1,23 @@
 #!python3
 
 # License: Apache 2.0. See LICENSE file in root directory.
-# Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
 
 import sys, os, subprocess, locale, re, platform, getopt
 from abc import ABC, abstractmethod
 
-# Add our py/ module directory to Python's list so we can use them
+# Remove Python's default list of places to look for modules!
+# We want only modules in the directories we specifically provide to be found,
+# otherwise pyrs other than what we compiled might be found...
+sys.path = list()
+sys.path.append( '' )

```

Comment:
Why do we need this?

Reply:
`''` (an empty string) is for the current directory... I'll add a comment

Reply:
Added `# directs Python to search modules in the current directory first` from [here](https://docs.python.org/3/library/sys.html#sys.path).